bindings/wasm/crate/src/lib.rs

```diff
@@ -29,7 +29,7 @@ pub fn create_outbound_session(
     sending_handle: &str,
     receiving_handle: &str,
     message: &str,
-) -> Result<String, JsValue> {
```

Comment:
I'm not sure I follow the need for the Box here.

My understanding was that JsValues through WASM_BINDGEN get copied between the memory space?

 Heap Allocating here seems unnecessary at a high level.